include/mgp.hpp

```diff
@@ -3276,6 +3278,53 @@ inline void Record::Insert(const char *field_name, const Duration &duration) {
   mgp::value_destroy(mgp_val);
 }
 
+inline void Record::Insert(const char *field_name, const Value &value) {
+  switch (value.Type()) {
+    case Type::Bool:
+      Insert(field_name, value.ValueBool());
+      break;
```

Comment:
Actually you can combine these two into return Insert, so please change that I gave approve too soon :joy:

Reply:
Done